PresnyaHedonist/Controllers/NoteListVC.swift

```diff
@@ -0,0 +1,140 @@
+//
+//  NoteListVC.swift
+//  PresnyaHedonist
+//
+//  Created by Alexandr L. on 7/13/21.
+//
+
+import UIKit
+import CoreData
+
+class NoteListVC: UIViewController {
+    
+    var place: Place?
+    var notes: [Note]?
+    
+    let appDelegate = UIApplication.shared.delegate as! AppDelegate
+    let context = (UIApplication.shared.delegate as! AppDelegate).persistentContainer.viewContext
+    var fetchedNotes: NSFetchedResultsController<Note>?
```

Comment:
Ah, to my question before I can see that you do have experience with NSFetchedResultsontroller! :-) Never mind, then. Or - what caused you to do the fetching manually in the other scenario? Was that the searching?

Reply:
Generally - not so acquainted with it, maybe core concepts. And about the previous one - the Search Bar still is pretty difficult for me and something I took from the tutorial that not uses CoreData. I know that I need learn more about both in any case)